addons/a11y/src/a11yRunner.ts

```diff
@@ -68,15 +70,20 @@ const run = async (storyId: string) => {
 
 /** Returns story parameters or default ones. */
 const getParams = (storyId: string): A11yParameters => {
-  const { parameters } = globalWindow.__STORYBOOK_STORY_STORE__.fromId(storyId) || {};
-  return (
-    parameters.a11y || {
-      config: {},
-      options: {
```

Comment:
restoreScroll does not exist in the interface for A11yParameters, nor could I find a reference to it in the code

anyone?

Reply:
It was passed straight through to axe, but it’s been gone there since 4.0: https://github.com/dequelabs/axe-core/issues/2388.